gravitee-am-service/src/main/java/io/gravitee/am/service/impl/ApplicationServiceImpl.java

```diff
@@ -787,6 +787,14 @@ private Single<Application> validateRedirectUris(Application application, boolea
                                 return Single.error(new InvalidRedirectUriException("redirect_uri : " + redirectUri + IS_MALFORMED));
                             }
                         }
+                        if(domain.isRedirectUriExpressionLanguageEnabled()){
+                            long uriWithELCount = oAuthSettings.getRedirectUris().stream()
+                                    .filter(uri -> new RedirectUri(uri).containsEL())
+                                    .count();
+                            if(uriWithELCount > 1) {
+                                return Single.error(new InvalidRedirectUriException("There can be only a single redirect_uri with EL"));
```

Comment:
this is not true, we can have multiple redirect URi with EL if the URI have different base URL.
ex:

* This is not valid as we do not know which one to pick
http://host1/path?query={#xxx}
http://host1/path?query={#yyy}

* This is valid as we can discrimiate
http://host1/path/one?query={#xxx}
http://host1/path/two?query={#yyy}
http://host2/path/two?query={#zzz}